tests/actions.go

```diff
@@ -516,9 +520,9 @@ func (oa *operatorActions) DeployOperatorOrDie(info *OperatorConfig) {
 func (oa *operatorActions) CleanOperator(info *OperatorConfig) error {
 	glog.Infof("cleaning tidb-operator %s", info.ReleaseName)
 
-	err := oa.CleanWebHookAndService(info.WebhookConfigName)
-	if err != nil {
-		return err
+	cmd := fmt.Sprintf("kubectl delete -f %s/webhook.yaml", oa.manifestPath(info.Tag))
+	if res, err := exec.Command("/bin/sh", "-c", cmd).CombinedOutput(); err != nil {
```

Comment:
instead, we should clean webhook in `manifests/webhook.yaml` deployed in the DeployOperator function.